rali/rali_pybind/amd/rali/plugin/tf.py

```diff
@@ -74,63 +73,78 @@ def __next__(self):
             self.loader.copyToTensorNCHW(self.out, self.multiplier, self.offset, self.reverse_channels, int(self.tensor_dtype))
         else:
             self.loader.copyToTensorNHWC(self.out, self.multiplier, self.offset, self.reverse_channels, int(self.tensor_dtype))
-                    
+        
         if(self.loader._name == "TFRecordReaderDetection"):
-            sum = 0
-            self.lis =[] #Empty list for bboxes
-            self.lis_lab=[] # Empty list of labels
-
-
-            for idx in range(self.bs):
-                sum=self.loader.GetBoundingBoxCount(idx)
-                self.labels = np.zeros(sum,dtype = "int32")
-                self.bboxes = np.zeros(sum*4,dtype = "float32" )
-                self.loader.GetBBLabels(self.labels,idx)
-                self.loader.GetBBCords(self.bboxes,idx)
+            num_bboxes = 0
```

Comment:
Codacy found an issue: [local variable 'num_bboxes' is assigned to but never used (F841)](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6302037)